app/presenters/document/field_group/section/value_presenter.rb

```diff
@@ -0,0 +1,143 @@
+# frozen_string_literal: true
+
+module Document
+  module FieldGroup
+    module Section
+      ##
+      # Presenter for one value of a metadata field
+      #
+      # TODO: should this be sub-classed? e.g. for single string values, language
+      #   maps, entity-linked values
+      class ValuePresenter < DocumentPresenter
```

Comment:
Sub classing might make sense for the different types of values, but more fundamentally, should we move the DocumentPreseneter logic into a concern? I don't know if it's just the semantics not quite making sense in my head, but 'DocumentPresenter' suggests to me that this is an object to Present an entire Document. 
These ValuePresenters or if we further sub-class them, EntityValuePresenters/LinkableValuePresenters/SimpleValuePresenters/whatver, do need to Present part of a Document, but don't display the whole Document.

Reply:
I agree in principle but the difficulty is that the methods for retrieving fields from documents are declared in `Blacklight::DocumentPresenter` (https://github.com/projectblacklight/blacklight/blob/v6.0.2/app/presenters/blacklight/document_presenter.rb) so wherever we use these, we need to have sub-classed that, here via `Europeana::Blacklight::DocumentPresenter` then `DocumentPresenter`.

Reply:
would an option be to make a concern that assigns the DocumentPresenter to an instance variable that can then be delegated to or used by our other presenters to pull the information from the document?
Along the lines of DocumentPresenterPresentingPresenter/DocumentPresenterUsingPresenter

If that was possible, the main downside would still be that our presenters for field_groups/sections/values will be dependant on the DocumentPresenter rather than being presenters for real models. Which isn't really the intend use of presenters as far as I can make out, although that's a bit of a grey area too. However this is inherent to the fact that we don't explicitly model FieldGroups/Sections/Values explicitly but rather derive these 'models' from the Document and our configuration.

Reply:
I have removed the sub-classing of `DocumentPresenter` except where it is absolutely necessary and introduced an `ApplicationController` base class for other presenters, which for now just delegates `#t` to `I18n`.